README.rst

```diff
@@ -59,6 +59,10 @@ CGC: Clustering Geo-Data Cubes
 Clustering Geo-Data Cubes (CGC) is a Python package to perform clustering analysis for multidimensional geospatial data.
 The included tools allow the user to efficiently run tasks in parallel on local and distributed systems.
 
+Statement of need
+-----------------
+The CGC package focuses on the needs of geospatial data scientists who require tools to make sense of multi-dimensional data cubes.
+
```

Comment:
This reads very nicely - I just suggest to incorporate it with the few lines before (now it feels a bit of a repetition) as in:
```suggestion
The Clustering Geo-Data Cubes (CGC) package focuses on the needs of geospatial data scientists who require tools to make sense of multi-dimensional data cubes. It provides the functionality to perform **co-cluster** and **tri-cluster** analyses on both local and distributed systems.
```
I think we can also drop the "statement of need" subtitle, it seems clear enough!